after_write_hooks/config/sqs.conf

```diff
@@ -1,11 +1,3 @@
-queues:
-    default:
-      - access_key_id: AWS_ACCESS_KEY_ID
-        secret_access_key: AWS_SECRET_ACCESS_KEY
-        region: us-west-2
-
-    sent_email:
-      - name: SQS_QUEUE_NAME
-        access_key_id: AWS_ACCESS_KEY_ID
-        secret_access_key: AWS_SECRET_ACCESS_KEY
-        region: us-west-2
+access_key_id: AWS_ACCESS_KEY_ID
```

Comment:
I also noticed that in the deploy repo we only have a single queue, so the sqs.conf can be simplified. I'm making a parallel change in the deploy repo in an upcoming PR for this file:

https://github.com/change/deploy/blob/master/site-cookbooks/change-fluentd/templates/default/sqs.conf.erb